src/Components/SchemaDotOrg/dataFromBlog.js

```diff
@@ -1,12 +1,15 @@
 import urlFromBinary from "../../utils/urlFromBinary";
+import isVideoObj from "../../utils/isVideoObj";
 
 function dataFromBlog(blog) {
-  return {
-    "@context": "http://schema.org",
-    "@type": "Blog",
-    headline: blog.get("title"),
-    image: urlFromBinary(blog.get("navigationBackgroundImage")),
-  };
+  if (!isVideoObj("navigationBackgroundImage")) {
```

Comment:
The context should still be rendered, even if the navigationBackgroundImage is a video:

```js
import isVideoObj from "../../utils/isVideoObj";
import urlFromBinary from "../../utils/urlFromBinary";

function dataFromBlog(blog) {
  const schema = {
    "@context": "http://schema.org",
    "@type": "Blog",
    headline: blog.get("title"),
  };

  const backgroundObj = blog.get("navigationBackgroundImage");
  if (backgroundObj && !isVideoObj(backgroundObj)) {
    schema.image = urlFromBinary(backgroundObj);
  }

  return schema;
}

export default dataFromBlog;
```